pennylane/math/quantum.py

```diff
@@ -262,7 +261,7 @@ def _batched_partial_trace(density_matrix, indices):
 

```

Comment:
Add a simple example without batching in the beginning. Then proceeed with a sentence like "Further, we can perform the ``partial_trace`` in a batched fashion:"

Reply:
added example!